.github/workflows/check.yaml

```
@@ -0,0 +1,65 @@
+name: Release
+
+on:
+  push:
+    branches-ignore:
+      - master
+
+env:
+  POETRY_CACHE_DIR: ".cache/poetry"
+  PIP_CACHE_DIR: ".cache/pip"
```

Should we keep these comments from the project-api-app repo ? There are several others that I think would be interesting.
Or maybe create a Notion article to have all these explanations in one place ?

```suggestion
env:
  # These environment variables are automatically recognized by poetry and pip
  # They allow us to specifically control where the cache is kept, so we can use 
  # the actions/cache@v1 action to store downloaded dependencies between multiple runs
  POETRY_CACHE_DIR: ".cache/poetry"
  PIP_CACHE_DIR: ".cache/pip"
```

---

Good idea. I'll add the Notion documentation to my todo.